cdap-distributions/src/etc/cdap/conf.dist/cdap-env.sh

```diff
@@ -38,6 +38,8 @@ LOCAL_DIR="/var/tmp/cdap"
 # temporary storage for Apache Twill
 TEMP_DIR="/tmp"
 
+HEAPDUMP_ON_OOM=true
```

Comment:
while it's technically not needed, it seems like convention that in hadoop *-env.sh files variables get exported...  So i guess we should also ``export`` them all, for consistency